modules/hal_nordic/nrfx/Kconfig.logging

```
@@ -0,0 +1,147 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menu "nrfx drivers logging"
+	depends on LOG
+
+config LOG_NRFX_ADC
+	bool "NRFX ADC driver logging"
```

```suggestion
	bool "ADC driver logging"
```
All of this is under the "nrfx drivers logging" menu so there is no need to repeat NRFX in each entry.